examples/regression/1-quickstart/plot_compare_conformity_scores.py

```diff
@@ -0,0 +1,197 @@
+# Two sub dataset are extracted: the training and test ones.
+
+X, y = fetch_openml(name="house_prices", return_X_y=True)
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X[features], y, test_size=0.2
+)
+
+##############################################################################
+# 2. Train model with two conformity scores
+# --------------------------------------------------------------------------
+#
+# Two models are trained with two different conformity score:
+# - AbsoluteConformityScore (default conformity score) relevant for target
+#   positive as well as negative. The confidence interval widths are in this
+#   case approximately the same over the range of prediction.
+# - GammaConformityScore relevant for target following roughly a Gamma
+#   distribution. The confidence interval widths scale with the predicted
+#   value.
+
+# Train model with AbsoluteConformityScore
+mapie = MapieRegressor(RandomForestRegressor(**rf_kwargs))
+mapie.fit(X_train, y_train)
+y_pred_absconfscore, y_pis_absconfscore = mapie.predict(X_test, alpha=[alpha])
+yerr_absconfscore = np.concatenate(
+    [
+        np.expand_dims(y_pred_absconfscore, 0) - y_pis_absconfscore[:, 0, 0].T,
+        y_pis_absconfscore[:, 1, 0].T - np.expand_dims(y_pred_absconfscore, 0),
+    ],
+    axis=0,
+)
+conf_int_width_absconfscore = (
+    y_pis_absconfscore[:, 1, 0] - y_pis_absconfscore[:, 0, 0]
+)
+coverage_absconfscore = regression_coverage_score(
+    y_test, y_pis_absconfscore[:, 0, 0], y_pis_absconfscore[:, 1, 0]
+)
+
+# Train model with GammaConformityScore
+mapie = MapieRegressor(
+    RandomForestRegressor(**rf_kwargs), conformity_score=GammaConformityScore()
+)
+mapie.fit(X_train, y_train)
+y_pred_gammaconfscore, y_pis_gammaconfscore = mapie.predict(
+    X_test, alpha=[alpha]
+)
+yerr_gammaconfscore = np.concatenate(
+    [
+        np.expand_dims(y_pred_gammaconfscore, 0)
+        - y_pis_gammaconfscore[:, 0, 0].T,
+        y_pis_gammaconfscore[:, 1, 0].T
+        - np.expand_dims(y_pred_gammaconfscore, 0),
+    ],
+    axis=0,
+)
+conf_int_width_gammaconfscore = (
+    y_pis_gammaconfscore[:, 1, 0] - y_pis_gammaconfscore[:, 0, 0]
+)
+coverage_gammaconfscore = regression_coverage_score(
+    y_test, y_pis_gammaconfscore[:, 0, 0], y_pis_gammaconfscore[:, 1, 0]
+)
+
+##############################################################################
+# 3. Compare the confidence intervals
+# --------------------------------------------------------------------------
+#
+# Once the models have been trained, we now compare the confidence intervals
+# obtained from the two conformity scores. We can see that the
+# AbsoluteConformityScore generates confidence interval with almost the same
+# width for all the predicted values. Converly, the GammaConformityScore
+# yields confidence interval with width scaling with the predicted values.
+#
+# The choice of the conformity score depends on the problem we face.
+
+fig, axs = plt.subplots(2, 2, figsize=(16, 16))
```

Comment:
Is is possible to decrease the size of the figure to increase the apparent font size in the generated example ?

Reply:
Done